src/main/java/Duke.java

```diff
@@ -1,10 +1,152 @@
+// import java.io.File; used for I/O redirection technique;
+import java.util.Scanner;
 public class Duke {
```

Comment:
Your Duke class is looking really long! Maybe consider breaking it up further into separate classes! That would help readability a lot!